### PR TITLE
Add lexoffice

### LINKED ISSUE #7688: Add Lexoffice
### Site name

Lexoffice

### Site URL

https://www.lexoffice.de/

### Documentation URL

https://support.lexoffice.de/de-form/articles/4290490

### Recovery Documentation URL

https://support.lexoffice.de/de-form/articles/5250370

### Supported 2FA methods

- [X] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] U2F/WebAuthn security keys
- [ ] Proprietary hardware (Yubico OTP, RSA keys, etc.)
- [ ] Phone calls
- [ ] SMS tokens
- [ ] Email tokens

### Regions served

Germany

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [X] The site is ranked within the top 200,000 global sites on [Similarweb](https://www.similarweb.com/).
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md).
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues) or [pull request](https://github.com/2factorauth/twofactorauth/pulls).

### Additional information

Category: Finance

### PR DESCRIPTION
**Pull request by kmpoppe**
Resolves #7688
Lexoffice Press office (https://www.lexoffice.de/presse/) only supplies the full logo (EPS) not an icon, so made that into an SVG.
Should an icon come along in the future this could be exchanged.

### CODE REVIEW

**Review by Carlgo11 — COMMENTED** [review]
Maybe a region tag?

File: img/l/lexoffice.de.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...39.14-101.964 106.91-101.964 48.48 0 85.15 26.105 90.74 67.754h113.14C2668.95 26.719 2584.39 0 2475.61 0c-125.56 0-210.72 53.469-210.72 179.023 0 125.551 88.28 179.004 220.06 179.004 84.53 0 183.36-25.476 203.88-135.492h-113.14M3150.06 111.875C3118.35 32.937 3042.52 0 2942.44 0c-131.76 0-220.03 53.438-220.03 179.004 0 125.57 88.27 179.031 220.03 179.031 144.21 0 213.83-72.738 213.83-198.277v-9.961h-320.73v-1.242c0-31.7 32.31-80.188 106.9-80.188 44.14 0 75.21 14.914 90.75 43.508zm-106.92 102.559v1.246c0 18.027-23 73.972-103.18 73.972-86.39 0-104.42-65.894-104.42-73.972v-1.246h207.6M851.504 95.156 934.98 11.72h167.93L934.98 178.652l-83.476-83.496" style="fill:#47484b;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/><path d="m600.102 345.605 167.933-166.953L600.102 11.688h167.933l83.469 83.496 83.484 83.468-166.953 166.953H600.102" style="fill:#f47218;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/><path d="m934.98 178.652 167.94 166.938H934.98l-83.472-83.465 83.472-83.473" style="fill:#47484b;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/></svg>
```
Comment (hkamran80) [inline img/l/lexoffice.de.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 67 44.5"><path fill="#47484b" d="m33.5 33.4 11.1 11.1H67L44.6 22.3l-11 11"/><path fill="#f47218" d="m0 0 22.4 22.3L0 44.5h22.4l11.1-11.1 11.2-11.1L22.4 0H0"/><path fill="#47484b" d="M44.6 22.3 67 0H44.6l-11 11.1 11 11.2"/></svg>
```

**Review by hkamran80 — COMMENTED** [review]
How about this SVG?

File: img/l/lexoffice.de.svg (hunk 1 above)
Comment (kmpoppe) [reply img/l/lexoffice.de.svg]: Can't see the suggested file, where did you get it from? (URL)

Comment (hkamran80) [reply img/l/lexoffice.de.svg]: I took the SVG you added and extracted the "x" from it

Comment (kmpoppe) [reply img/l/lexoffice.de.svg]: Smart thinking, thank you!

Comment (hkamran80) [reply img/l/lexoffice.de.svg]: You're welcome!

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**